LINVAST.Imperative/Builders/Java/JavaASTBuilder.Types.cs

```diff
@@ -14,6 +14,193 @@ namespace LINVAST.Imperative.Builders.Java
 {
     public sealed partial class JavaASTBuilder : JavaBaseVisitor<ASTNode>, IASTBuilder<JavaParser>
     {
-        // TODO after types are added to AST
+        public override ASTNode VisitTypeDeclaration([NotNull] TypeDeclarationContext ctx)
+        {
+
```

Comment:
Extra space. Please remove this and other unnececary spaces.

Reply:
I tried with visual studio formatting I think that this remark is resolved.